### ISSUE TITLE
variable name format bug

### ISSUE THREAD

**Issue by mmase** [sasstools/sass-lint#901]
The following throws a warning for variable name format, when it shouldn't (likely due to no spaces between variable name and operator):

```scss
.content-main {
  padding: $mobile-site-gutter*1.5;
}
```

**Comment by DanPurdy** [comment]
@mmase Thanks for this. Looks like an issue with the AST we use reporting the variable name incorrectly. I've put a PR into sass-lint though which should get around this problem for now. Expect to see it fixed in 1.10
